corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html

```diff
@@ -102,27 +102,31 @@
       <div class="d-lg-none">
         <div id="mobile-restore-as-region"></div>
       </div>
-      <div id="breadcrumb-region" class="print-container sticky-top"></div>
```

Comment:
Does this make breadcrumbs no longer sticky on scroll?  Or is it that the whole top part of the screen (navbar and breadcrumbs) doesn't scroll at all?  With this change and with making everything a vertical flex element, does that mean we don't need to do the `.case-tile-container` and `.query-button-container` offset stuff anymore?

Reply:
Ah, I see you did remove the offset for the latter already.

Reply:
superb